examples/Reading in linearised models.ipynb

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
<p>Convert these to headings with # (throughout the notebook)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jessegrabowski/gEconpy/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>